test/foundry/apps.t.sol

```diff
@@ -749,4 +750,187 @@ contract FleekTest is Test {
 
         assertEq(fleekContract.balanceOf(DEPLOYER), 1);
     }
+
+    function testAddAccessPoint() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
```

Comment:
Maybe use a variable for the domain because I see it's repeated a lot in the file.

Reply:
agree.